Framework/Algorithms/inc/MantidAlgorithms/ApplyInstrumentToPeaks.h

```diff
@@ -0,0 +1,32 @@
+// Mantid Repository : https://github.com/mantidproject/mantid
+//
+// Copyright &copy; 2021 ISIS Rutherford Appleton Laboratory UKRI,
+//   NScD Oak Ridge National Laboratory, European Spallation Source,
+//   Institut Laue - Langevin & CSNS, Institute of High Energy Physics, CAS
+// SPDX - License - Identifier: GPL - 3.0 +
+#pragma once
+
+#include "MantidAPI/Algorithm.h"
+#include "MantidAlgorithms/DllConfig.h"
+namespace Mantid {
+namespace Algorithms {
+
+/** ApplyInstrumentToPeaks : TODO: DESCRIPTION
+ */
+class MANTID_ALGORITHMS_DLL ApplyInstrumentToPeaks : public API::Algorithm {
+public:
+  const std::string name() const override { return "ApplyInstrumentToPeaks"; }
+  int version() const override { return 1; };
+  const std::vector<std::string> seeAlso() const override { return {}; }
+  const std::string category() const override { return "Crystal\\Peaks"; }
+  const std::string summary() const override {
+    return "Update the peaks within a PeaksWorkspace with a different instrument, keeping the same detectorID and TOF";
```

Comment:
How about changing the summary to "Update the instrument attached to peaks to match the one provided or 
 the one attached to the input workspace while keeping detectorID and TOF unchanged"?